static/src/javascripts-legacy/projects/common/modules/identity/delete-account.js

```diff
@@ -5,10 +5,19 @@ define([
     $,
     bean
 ) {
+    function disableDeleteButton() {
+        $('#deleteButton')[0] && ($('#deleteButton')[0].disabled = true);
+    }
+
+    function showLoader() {
+        $('#deleteLoader')[0] && $('#deleteLoader')[0].classList.remove("is-hidden");
+    }
+
     function setupLoadingAnimation() {
-        if ($('#deleteForm').length && $('#deleteLoader').length) {
+        if ($('#deleteForm')[0] && $('#deleteLoader')[0]) {
             bean.on($('#deleteForm')[0], 'submit', function() {
-                $('#deleteLoader')[0].classList.remove("is-hidden");
+                disableDeleteButton();
+                showLoader();
```

Comment:
Can we wrap the calls to `disableDeleteButton` and `showLoader` in a `fastdom.write` callback? We always try to wrap DOM manipulation statements in `fastdom` to ensure it happens as smoothly as possible i.e.

```
fastdom.write(function() {
    disableDeleteButton();
    showLoader();
});
```

[Example](https://github.com/guardian/frontend/blob/master/static/src/javascripts-legacy/projects/common/modules/identity/email-preferences.js#L111-L117)